contracts/WooFeeManager.sol

```diff
@@ -59,87 +60,69 @@ contract WooFeeManager is InitializableOwnable, ReentrancyGuard, IWooFeeManager
 
     /* ----- State variables ----- */
 
-    mapping(address => uint256) public override feeRate;
+    mapping(address => uint256) public override feeRate; // decimal: 18; 1e16 = 1%, 1e15 = 0.1%, 1e16 = 0.01%
     address public wooPP;
-    address public quoteToken;
-    address public rewardToken;
-    IWooRewardManager public rewardManager;
+    address public immutable quoteToken;
+    IWooRebateManager public rebateManager;
+    IWooVaultManager public vaultManager;
+    uint8 private vaultRewardPercent; // in uint 100th. 1 means 1%, 100 means 100%
```

Comment:
Maybe using the usual 10^18 representation is more uniform. I don't think the current version saves any space or gas. I'm neutral to the change, because the current version is easier to understand.

Reply:
Okay . updating.